code/modules/antagonists/heretic/heretic_antag.dm

```diff
@@ -559,6 +561,12 @@
 /datum/antagonist/heretic/proc/get_knowledge(wanted)
 	return researched_knowledge[wanted]
 
+/datum/antagonist/heretic/proc/increase_rust_strength(side_path_only = FALSE)
```

Comment:
Consider making a setter for this as well - so you can directly set strength to a certain level (r-wall level or something)